src/logic/apiRequests.js

```diff
@@ -28,6 +25,10 @@ export const deleteReservation = (id, equipmentId, date) => async (dispatch) =>
     return res.data;
   })
     .catch((err) => {
+      if (err.response?.status === 500) {
```

Comment:
500 status is for Internal Server Error, let's make it 401 for Unauthorized errors please.